app/pages/notifications/index.jsx

```diff
@@ -12,7 +12,7 @@ counterpart.registerTranslations('en', {
     header: 'Notifications',
     title: 'My Notifications',
     signedOut: 'You\'re not signed in.',
-    noNotifications: 'You have no notifications.',
+    noNotifications: 'You have no notifications. ',
```

Comment:
Best to put the space between the translated text strings here instead.
https://github.com/zooniverse/Panoptes-Front-End/blob/b59be6cd8c20054534722dc78d221d05c33ec0c7/app/pages/notifications/index.jsx#L113

Reply:
This has broken the test, which explain why Travis is failing.